torchci/lib/types.ts

```diff
@@ -18,7 +18,7 @@ export interface JobData {
   queueTimeS?: number;
   failureLine?: string;
   failureLineNumber?: number;
-  failureCaptures?: string;
```

Comment:
`failureCaptures` should be a list of string instead, so this corrects the type declaration.  This is kind of unrelated to the PR, but `yarn tsc` surfaces that error, so I think I would rather fix it right away.